lib/code_corps/github/api.ex

```diff
@@ -0,0 +1,28 @@
+defmodule CodeCorps.Github.API do
+  @client_secret Application.get_env(:code_corps, :github_client_secret)
+  @client_id Application.get_env(:code_corps, :github_client_id)
+
+  @base_connect_params %{
+    client_id: @client_id,
+    client_secret: @client_secret
+  }
+
+  def connect(code) do
+    with {:ok, %HTTPoison.Response{body: response}} <- code |> build_connect_params() |> do_connect(),
```

Comment:
I had issues getting it to work. The documented methods do not work as documented. They also raise exceptions when they should not be doing so and are named without exclamation marks.

Due to that, for this specific case, I went with a direct API call.

For the regular stuff, in the integration milestone, we should go with Tentacat.